world.js

```diff
@@ -94,173 +89,182 @@ const _bindState = (state, dynamic) => {
+        buffArgs(
+          extra.leftGamepad,
+          [
+            leftGamepad.pointer ? 1 : 0,
+            leftGamepad.grip ? 1 : 0,
+            leftGamepad.enabled ? 1 : 0
+          ],
+        ),
+        buffVec(extra.rightGamepadPosition, rightGamepad.position),
+        buffVec(extra.rightGamepadQuaternion, rightGamepad.quaternion),
+        buffArgs(
+          extra.rightGamepad,
+          [
+            rightGamepad.pointer ? 1 : 0,
+            rightGamepad.grip ? 1 : 0,
+            rightGamepad.enabled ? 1 : 0,
+          ]
+        ),
+        buffArgs(
+          extra.attr,
+          [
+            rig.getFloorHeight() ? 1 : 0,
+            rig.getTopEnabled() ? 1 : 0,
+            rig.getBottomEnabled() ? 1 : 0,
+          ]
+        ),
+        buffVec(extra.direction, rig.direction),
+        buffVec(extra.velocity, rig.velocity),
+        buffArgs(
+          extra.states,
+          [
+            rig.jumpState,
+            rig.jumpTime,
+            rig.flyState,
+            rig.flyTime,
+            rig.useTime,
+            rig.useAnimation,
+            rig.sitState,
+            rig.sitAnimation,
+            rig.danceState,
+            rig.danceTime,
+            rig.danceAnimation,
+            rig.throwState,
+            rig.throwTime,
+            rig.crouchState,
+            rig.crouchTime,
+          ]
+        ),
+      ],
+    );
+  };
+
+  wsrtc.addEventListener('open', async e => {
     console.log('Channel Open!');
 
-    if (networkMediaStream) {
-      channelConnection.setMicrophoneMediaStream(networkMediaStream);
-    }
-    
-
-    interval = setInterval(() => {
-      const name = loginManager.getUsername();
-      const avatarSpec = loginManager.getAvatar();
-      const avatarUrl = avatarSpec && avatarSpec.url;
-      const avatarExt = avatarSpec && avatarSpec.ext;
-      const address = loginManager.getAddress();
-      const aux = rigManager.localRig?.aux.getPose();
-      channelConnection.send(JSON.stringify({
-        request: true,
-        id: Math.floor(Math.random() * Number.MAX_SAFE_INTEGER),
-        method: 'status',
-
-        data: {
-          peerId: channelConnection.connectionId,
-          status: {
-            name,
-            avatarUrl,
-            avatarExt,
-            address,
-            aux,
-          },
-        },
-      }));
-      const pose = rigManager.getLocalAvatarPose();
-      channelConnection.send(JSON.stringify({
-        request: true,
-        id: Math.floor(Math.random() * Number.MAX_SAFE_INTEGER),
-        method: 'pose',
-        data: {
-          pose,
-        },
-      }));
-    }, 10);
-  
+    const name = makeId(5);
+    wsrtc.localUser.setMetadata({
+      name,
+    });
 
-    /* world.dispatchEvent(new MessageEvent('peersupdate', {
-      data: Array.from(rigManager.peerRigs.values()),
-    })); */
+    interval = setInterval(sendUpdate, 10);
+    wsrtc.enableMic();
```

Comment:
Note that we had the `T` key for talk before, but it seems like that's changing now.

Reply:
the T key should still toggle but I can have it not autoenable.

Reply:
No it's ok, this actually makes more sense. We'll be deleting the old UI code anyway as we hook up React.